setup.cfg

```diff
@@ -13,62 +13,59 @@ license = Apache-2.0
 classifiers =
   Development Status :: 4 - Beta
   Intended Audience :: Developers
-  Programming Language :: Python :: 3.6
+#   Programming Language :: Python :: 3.6 ## numpy >= 1.20.0 dropped support for python 3.6
   Programming Language :: Python :: 3.7
   Programming Language :: Python :: 3.8
   Topic :: Scientific/Engineering :: Artificial Intelligence
   Topic :: Scientific/Engineering :: Image Recognition
 
 [options]
-python_requires = >=3.6,<4
+python_requires = >=3.7,<4
 zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    pillow > 8.0.0,<9
+    pillow >8.0.0,<9
     torch >=1.9.0,<1.11
     torchvision >=0.10.0,<0.12
     fastcore >=1.3.0,<1.4
     tqdm >=4.49.0,<5
     opencv-python >=4.1.1,<5
     albumentations >=1.0.3,<1.1
-    matplotlib >=3.2.2,<4
-    pycocotools>=2.0.2,<3
-    requests >=2.23.0,<3
+    resnest >=0.0.6b20201125,<0.0.7
+    effdet >=0.2.1,<0.3
+    sahi >=0.8.19,<1.0
+    yolov5-icevision >=6.0.0
+    importlib-metadata >=1;python_version<"3.8"
+    ipykernel >=4.10.1,<6
+    dataclasses ==0.6
     loguru >=0.5.3
-    importlib-metadata>=1;python_version<"3.8"
+
 
 [options.extras_require]
+inference =
+    icevision
```

Comment:
Why did this change?

Reply:
I am actually a bit confused by the changes in this entire file. Shall we move the discussion to Discord?